h264/sps.go

```diff
@@ -531,34 +515,17 @@ func NewSPS(rbsp []byte, showPacket bool) (*SPS, error) {
 			if v := b.NextField("MotionVectorsOverPicBoundaries", 1); v == 1 {
 				sps.MotionVectorsOverPicBoundaries = true
 			}
-			sps.MaxBytesPerPicDenom, err = readUe(nil)
```

Comment:
@kortschak I'm trying to reduce the verbosity of a series of readUe calls. See the change I made here using readUes. I need to give references to the fields, or is this not worth it ?